enterprise/reporting/error_index/worker_test.go

```diff
@@ -458,7 +458,7 @@ func TestWorkerWriter(t *testing.T) {
 				}
 				p.SetReceivedAt(receivedAt)
 				p.SetFailedAt(failedAt.Add(time.Duration(i) * time.Second))
-				payloads = append(payloads, p)
+				_ = append(payloads, p)
```

Comment:
We can also remove `payloads := make([]payload, 0, count)`. It's not being used
```suggestion
```

Reply:
Already did this